wpilib/wpilib/analogtriggeroutput.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, trigger, outputType):
         :param outputType: An enum that specifies the output on the trigger
             to represent.
         """
+        super().__init__()
```

Comment:
This has nothing to do with type hinting.

Reply:
Huh I thought I got them all